### ISSUE TITLE
Site and Docs are down?

### ISSUE THREAD

**Issue by NanyThery** [metafizzy/flickity#1286]
**Test case:** https://codepen.io/desandro/pen/azqbop

Guys, Not sure if this is something isolated, but I cannot access any of your websites to check the docs. Is this something permanent?:

![image](https://github.com/metafizzy/flickity/assets/47493473/e537ba3f-9827-4a88-9b47-d1e0ddc084d3)

![image](https://github.com/metafizzy/flickity/assets/47493473/27bb0b1b-e5d4-44cd-953b-5f92d1b11b8b)

**Comment by haroldao** [comment]
same issue :/

**Comment by desandro** [comment]
Yes, this indeed an issue. Netlify has spuriously banned my account, which has taken down a dozen of my sites. I am now working to move hosting provider. This will likely take another couple days to resolve.

Sorry for the hassle everyone

More drama detail in case you're curious:

Netlify has completely banned my account for "fraudulent behavior." I have literally no idea in reference to what. Uncertain what recourse I have with them. I may need to re-deploy a dozen sites elsewhere. I am absolutely furious

1. I took 72 hours through a support thread for them to reveal the ban.
2. This is the first mention about fraudulent behavior. No previous correspondence about it
3. Most recent successful payment was Oct 6th

**Comment by haroldao** [comment]
ty for the update and sorry to hear that! @desandro

**Comment by NanyThery** [comment]
Sorry to hear that, thank you for the reply. I understand that you are furious, is a lot of extra work and poor customer service. I hope you can sort this out soon. And by the way, thank you for this library and the support 👍🏼

**Comment by marcomezzavilla** [comment]
What a useless waste of time. Netlify's behavior seems very unfair to me.
